extension/src/project/projectCommands.ts

```diff
@@ -212,6 +213,16 @@ export function registerProjectCommands(context: vscode.ExtensionContext) {
             }
         }));
 
+        context.subscriptions.push(vscode.commands.registerCommand('autolisp.openWebHelp', () => {
+            try {
+                openWebHelp();
+            }
+            catch (err) {
+                if (err)
+                    console.log(err.toString());
+            }
+        }));
```

Comment:
```suggestion
    context.subscriptions.push(vscode.commands.registerCommand('autolisp.openWebHelp', async () => {
            try {
                await openWebHelp();
            }
            catch (err) {
                if (err)
                    console.log(err.toString());
            }
        }));
```
Use async/await to ensure we can call async apis within `openWebHelp()`